website/routes.py

```diff
@@ -227,6 +228,20 @@ def make_url_map(app):
         ),
 
         Rule('/news/', 'get', {}, OsfWebRenderer('public/pages/news.mako')),
+
+        Rule(
+            '/prereg/',
+            'get',
+            prereg.prereg_landing_page,
+            OsfWebRenderer('prereg_landing_page.mako', trust=False)
+        ),
+
+        Rule(
+            '/api/v1/prereg/draft_registrations/',
```

Comment:
I don't know. I *think* @samanehsan might have worked on that, but it was a long time ago and I never did anything with it directly. That could also have been a project that was only done to prototype stage, though I think it made it in.